kuma/users/jinja2/users/includes/stripe_subscription.html

```diff
@@ -0,0 +1,76 @@
+<noscript>
+  <style>
+    .stripe-subscription {
+      display: none
+    }
+  </style>
+</noscript>
+
+<section class="stripe-subscription">
+  {% stylesheet 'stripe-subscription' %}
+
+  {% if not subscription_info %}
+    <form id="stripe-form" action="{{ url('users.create_stripe_subscription') }}" method="post">
+      {% csrf_token %}
+      <h2>Help grow MDN</h2>
```

Comment:
My assumption was that since this is a UI that is bound to change and for this stage only planned for internal use, it wouldn't need translation. Wdyt @atopal?

Reply:
For the initial launch it's not necessary, but at the same time, there is no intention to change this implementation any time soon. We'll add more obvious and upfront messaging, but this implementation in the profile would stay.